djasana/management/commands/sync_from_asana.py

```diff
@@ -307,6 +310,7 @@ def _sync_task(self, task, project, models, skip_subtasks=False):
         For parents and subtasks, this method is called recursively, so skip_subtasks True is
         passed when syncing a parent task from a subtask.
         """
+        was_synced = []
```

Comment:
In my mind, variables is_* and was_* are expected to be boolean, like is_archived. Changing this variable to synced_ids conforms to the style I've used.

Reply:
great point